src/wireviz/wv_helper.py

```diff
@@ -53,6 +56,41 @@ def nested_html_table(rows):
     html.append('</table>')
     return html
 
+def html_colorbar(color):
+    return f'<tdX bgcolor="{wv_colors.translate_color(color, "HEX")}" width="4">' if color else None
+
+def html_image(node):
+    if not node.image:
+        return None
+    size = html_size_attr(node.image_size)
+    # The leading attributes belong to the preceeding tag. See where used below.
+    html = f'{"".join(size)}><img scale="{node.image_scale}" src="{node.image}"/>'
+    if size[2]: # fixedsize
+        # Close the preceeding tag and enclose the image cell in a table without
+        # borders to avoid narrow borders when the fixed width < the node width.
+        html = f'''>
+    <table border="0" cellspacing="0" cellborder="0"><tr>
+     <td{html}</td>
+    </tr></table>
+   '''
+    return f'''<tdX{' sides="TLR"' if node.caption else ''}{html}'''
+
+def html_caption(node):
+    return f'''<tdX{' sides="LRB"' if node.image else ''}>{html_line_breaks(node.caption)}''' if node.caption else None
+
+def html_size_attr(size):
+    # Return Graphviz HTML attributes to specify minimum size of a TABLE or TD object
+    # size: List of values where only these are used:
+    # - First value is minimum width of object in points, an int value in the range 1-65535. (Default 0 = none)
+    # - Second value is minimum height of the object in points, an int value in the range 1-65535. (Default 0 = none)
+    # - Third value is a boolean where TRUE specify that the first two values are the fixed image cell size.
+    #   (Default FALSE) Other string values, e.g. FIXED are also interpreted as TRUE.
```

Comment:
Does such a small function require such a long comment? The goal should be to have code that is understandable enough that most comments are unnecessary, maybe there is potential for simplification here... The int range is interesting, but maybe irrelevant since it's unlikely somebody will really need an image 65535 px wide, so it's a tradeoff between documenting for edge cases, vs. readability.
What do you think?

Reply:
The value type, limit, and default and are copied from Graphviz doc. Should we rather link to that?

Reply:
A small note to the GraphViz doc should be enough, yes, it's a good compromise :)

Reply:
PS: Please don't mark conversations as "resolved" when there are open questions, otherwise they might go unnoticed. I only resolve when there is no disagreement at all and the change is implemented exactly according to the comment.